app/src/main/java/com/cornellappdev/uplift/ui/screens/ProfileCreationScreen.kt

```diff
@@ -0,0 +1,227 @@
+                        fontWeight = FontWeight.Normal
+                    )
+                }
+
+                Row(verticalAlignment = Alignment.CenterVertically) {
+                    Checkbox(
+                        checked = checkedState2.value,
+                        onCheckedChange = { checkedState2.value = it },
+                        colors = checkboxColors
+                    )
+                    Text(
+                        text = "I allow Uplift to access data on my gym usage",
+                        fontSize = 12.sp,
+                        fontFamily = montserratFamily,
+                        fontWeight = FontWeight.Normal
+                    )
+                }
+
+                Row(verticalAlignment = Alignment.CenterVertically) {
+                    Checkbox(
+                        checked = checkedState3.value,
+                        onCheckedChange = { checkedState3.value = it },
+                        colors = checkboxColors
+                    )
+                    Text(
+                        text = "I allow Uplift to access my location",
+                        fontSize = 12.sp,
+                        fontFamily = montserratFamily,
+                        fontWeight = FontWeight.Normal
+                    )
+                }
+
+            }
+
+            if (checkedState.value && checkedState2.value && checkedState3.value) {
+                Spacer(modifier = Modifier.height(155.dp))
+
+                Row(
+                    verticalAlignment = Alignment.CenterVertically,
+                    modifier = Modifier.padding(bottom = 25.dp)
+                ) {
+                    Text(
+                        text = "Are you ready to",
+                        fontSize = 16.sp,
+                        fontFamily = montserratFamily,
+                        fontWeight = FontWeight.Bold
+                    )
+
+                    Spacer(modifier = Modifier.width(6.dp))
+
+                    Image(
+                        painter = painterResource(R.drawable.ic_main_logo),
+                        contentDescription = "",
+                        modifier = Modifier.size(width = 26.dp, height = 23.dp)
+                    )
+
+                    Image(
+                        painter = painterResource(R.drawable.lift_logo),
+                        contentDescription = "",
+                    )
+                    Spacer(modifier = Modifier.width(6.dp))
+
+                    Text(
+                        text = "?",
+                        fontSize = 16.sp,
+                        fontFamily = montserratFamily,
+                        fontWeight = FontWeight.Bold
+                    )
+                }
+            } else {
+                Spacer(modifier = Modifier.height(205.dp))
+            }
+            Button(
+                {},
+                enabled = checkedState.value && checkedState2.value && checkedState3.value,
```

Comment:
Hmm for the time being let's pull this out into a variable at the top called `val allChecked = checkedState.value && checkedState2.value && checkedState3.value`.

For best practices in the end, you'll actually want the viewmodel to handle all this checked logic, so let's avoid doing this && computation several times and instead reference some value. Then you'll be able to change this value to what the viewmodel says.